src/common/options/crimson.yaml.in

```diff
@@ -105,8 +105,8 @@ options:
 - name: seastore_max_data_allocation_size
   type: size
   level: advanced
-  desc: Max size in bytes that an extent can be
-  default: 32_K
+  desc: Max size in bytes that an extent can be, 0 to disable
+  default: 0
```

Comment:
No reason to keep this option with beside nice-to-have, right?

Reply:
We still need it, otherwise the read amplification can be very large in scenarios where full extent integrity check is needed.

Reply:
Right, CRC is still validated at the extent granularity, not 4K-grained. So when 4KB is loaded from a 4MB extent, CRC can't be checked.